### PR TITLE
completed docker setup and created docker image and container

### PR DESCRIPTION
**Pull request by harmouche**
As suggested by stakeholders, I setup Docker image and container to use the code environment universally.

### CODE REVIEW

**Review by NicoleYOsborn — CHANGES_REQUESTED** [review]
It looks like the test is just failing due to a missing semicolon on line 3 of server.js.

File: server.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const app = require('./api/app.js');
 
-const port = process.env.PORT || 8000;
+const port = process.env.PORT || 8080
```
Comment (NicoleYOsborn) [inline server.js]: So this is how you leave a comment on a particular part of the code!  This is where it shows a missing semicolon. :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+ENV OKTA_URL_ISSUER="https://auth.lambdalabs.dev/oauth2/default"
+ENV OKTA_CLIENT_ID=0oa18is3355KlyP5C4x7
+ENV NODE_ENV="production"
+ENV PORT=8080
+
+EXPOSE 8080
+
+CMD [ "npm", "start" ]
```
Comment (NicoleYOsborn) [inline Dockerfile:19]: Thank you for setting this up! Hopefully, we can meet up this weekend so I can see how it works.

**Review by harmouche — COMMENTED** [review]
fixed the bug, added ";" to line 3 in server.js

File: Dockerfile (hunk 2 above)
Comment (harmouche) [reply Dockerfile:19]: Yeah, docker will help us a lot with this project.

File: server.js (hunk 1 above)
Comment (harmouche) [reply server.js]: I took care of it.

**Review by NicoleYOsborn — APPROVED** [review]
Great!

**Pull request merged**